src/realm/error_codes.cpp

```diff
@@ -393,6 +393,8 @@ static const constexpr MapElem string_to_error_code[] = {
     {"SyncConnectFailed", ErrorCodes::SyncConnectFailed},
     {"SyncConnectTimeout", ErrorCodes::SyncConnectTimeout},
     {"SyncInvalidSchemaChange", ErrorCodes::SyncInvalidSchemaChange},
+    {"SyncLocalClockBeforeEpoch", ErrorCodes::SyncLocalClockBeforeEpoch},
```

Comment:
This was never added to the `string_to_error_code` array.

Reply:
💯

Reply:
shouldn't the new errors be added to this map? and also assign error categories?